Classes/Helper/PathSegmentHelper.php

```diff
@@ -27,20 +27,19 @@ class PathSegmentHelper
      */
     protected $slugHelper;
 
-    public function __construct(SlugHelper $slugHelper = null)
```

Comment:
Just for your information: I used it that way for better UnitTesting. But it's OK so. We still can overwrite instantiation in GeneralUtility::addInstance()